cmd/admin-handlers-site-replication.go

```diff
@@ -543,3 +544,27 @@ func (a adminAPIHandlers) SiteReplicationResyncOp(w http.ResponseWriter, r *http
 	}
 	writeSuccessResponseJSON(w, body)
 }
+
+// SiteReplicationDevNull - everything goes to io.Discard
+// [POST] /minio/admin/v3/site-replication/devnull
+func (a *adminAPIHandlers) SiteReplicationDevNull(w http.ResponseWriter, r *http.Request) {
+	globalNetPerfRX.Connect()
```

Comment:
don't re-use the `netperf`s global value instead allocate a new one indicating in globals.go

```
globalSiteNetPerfRX netPerfRX
```